exercises/pangram/canonical-data.json

```
@@ -94,6 +94,15 @@
         "sentence": "the quick brown fox jumps over with lazy FX"
       },
       "expected": false
+    },
+    {
+      "uuid": "7138e389-83e4-4c6e-8413-1e40a0076951",
+      "description": "a-m and A-M are 26 different letters but not a pangram",
```

```suggestion
      "uuid": "7138e389-83e4-4c6e-8413-1e40a0076951",
      "reimplements": "2577bf54-83c8-402d-a64b-a2c0f7bb213a",
      "description": "a-m and A-M are 26 different letters but not a pangram",
```

Would this make sense?

The sentence for that test case is pretty weird:
> "the quick brown fox jumps over with lazy FX"

and makes the reader wonder about "with" and "FX".

So if we think they test the same thing, we could just replace it.

---

I like that.